tests/e2e/performance/test_pvc_creation_performance.py

```diff
@@ -24,6 +24,32 @@ class TestPVCCreationPerformance(E2ETest):
     """
     pvc_size = '1Gi'
 
+    def samples_creation(self, samples_num, teardown_factory, pvc_size):
```

Comment:
shouldn't the function be called 'measure_multiple_pvcs_creation_time`

Reply:
@ebenahar 
I agree that the name should be more descriptive. 
Let's do it "create_mutiple_pvcs_statistics "?
Otherwise it can be easily seen as a duplication of tests.helpers.measure_pvc_creation_time .

Reply:
sounds good